lib/polaris_form_builder.rb

```diff
@@ -0,0 +1,90 @@
+class PolarisFormBuilder < ActionView::Helpers::FormBuilder
+  def polaris_text_field(method, options = {}, &block)
+    help_text = options.delete(:help)
+    
+    @template.content_tag(:div, class: "Polaris-FormLayout__Item") do
+      @template.safe_join([
+        @template.content_tag(:div, class: "Polaris-Labelled__LabelWrapper") do
+          @template.content_tag(:div, class: "Polaris-Label") do
+            label(method, class: "Polaris-Label__Text")
+          end
+        end,
+
+        @template.content_tag(:div, class: "Polaris-Connected") do
+          @template.content_tag(:div, class: "Polaris-Connected__Item Polaris-Connected__Item--primary") do
+
+            classes = ["Polaris-TextField"]
+            classes << "Polaris-TextField--error" if @object && @object.errors.details[method.to_sym].present?
+            
+            @template.content_tag(:div, class: classes) do
+              @template.safe_join([
+                text_field(method, options.merge(class: "Polaris-TextField__Input")),
```

Comment:
This seems to still add the error wrapper. 

While the UI still show the `"Polaris-TextField--error"`, because of the extra div it does not apply the CSS.  

<img width="1194" alt="Screen Shot 2022-02-25 at 12 56 59 PM" src="https://user-images.githubusercontent.com/207902/155764075-02cb1af3-d66a-432c-8260-11d18e23a8c0.png">

Reply:
I forgot to commit the new form_error_proc. 😅